quick_start.adoc

```diff
@@ -282,6 +283,63 @@ oc create -f <authrealm_file_name>.yaml
 At this point, any managed cluster that matches your Placement should be updated with an OAuth configuration that leverages an OpenID identity provider to connect to the proxy server that identity configuration management has stood up under the covers. Allow several minutes for the new login button to appear on your managed cluster.
 You will be able to login with the ldap user.
 
+[#openid-authentication]
+OpenID identity provider configuration provides identities for users authenticating using OpenID credentials. You can configure Red Hat Single Sign-On as an OpenID Connect identity provider.
+
+=== AuthRealm custom resource for OpenID
+An example of an AuthRealm custom resource configured for the OpenID identity provider is provided below. Make note of the `identityProviders` field and the configuration of the `opeinid` identity provider under it:
+[source,yaml]
+----
+apiVersion: identityconfig.identitatem.io/v1alpha1
+kind: AuthRealm
+metadata:
+  name: authrealm-openid
+  namespace: authrealm-openid-ns
+spec:
+  placementRef:
+    name: dev-clusters-placement
+  routeSubDomain: identitatem-devs
+  type: dex
+  identityProviders:
+    - name: identitatem-developers
+      mappingMethod: add
+      type: OpenID
+      openID:
+        clientID: 41c30d83fd1b4774f871
+        clientSecret:
+          name: identitatem-openid-client-secret  
+        claims:
+          preferredUsername:
+            - username
+          name:
+            - name
+          email:
+            - mail
+----
+
+The `identityProviders` list contains the configurations for one or more identity providers. The example above contains a single identity provider (OpenID).
+An entry under `identityProviders` has the following fields:
+
+- `name` contains the unique name that is used to identify the identity provider.
+- `type` specifies the identity provider type and it is set to OpenID.
+- `mappingMethod` (add, claim or lookup) controls how mappings are established between this provider’s identities and User objects.
+- `openID` contains the OpenID specific configurations:
+  * `clientID` contains the client ID of OAuth.
```

Comment:
"client ID of OAuth" > "client ID of a client registered with the OpenID provider"?